src/lxc/tools/lxc_attach.c

```diff
@@ -333,6 +345,14 @@ int main(int argc, char *argv[])
 			goto out;
 	}
 
+	if (my_args.uid) {
```

Comment:
Unnecessary braces here and in the gid case.

Reply:
done